src/Apps/Auction/Components/Form/IdentityVerificationWarning.tsx

```diff
@@ -22,7 +24,7 @@ export const IdentityVerificationWarning: React.FC = () => {
         <RouterLink color="black100" to="/conditions-of-sale" target="_blank">
           Conditions of Sale
         </RouterLink>
-        .
```

Comment:
**non-clocking/question**: if `additionalText` is `undefined`, does it still look okay in the UI? I'm surprised that TS is not asking for a truthy check for it.

Reply:
Added a few more screenshots. TS is not asking for a truthy check because `additionalText` is not a required property, so this prop can be skipped or has `undefined` value which is the same. But I decided to use empty string instead of undefined value just in case.